test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -54,82 +89,6 @@ func TestSchedule100Node3KPods(t *testing.T) {
 	}
 }
 
-// TestSchedule100Node3KNodeAffinityPods schedules 3k pods using Node affinity on 100 nodes.
-func TestSchedule100Node3KNodeAffinityPods(t *testing.T) {
```

Comment:
@ravisantoshgudimetla true .......... But I thought we are **keeping the test**, and just make it a thin wrapper?  We want test results to be identical ...